transformer/events_creator.go

```diff
@@ -77,17 +78,20 @@ func (c *Transformer) createBeatEvents(fetchedResource fetching.Resource, cycleM
 	fetcherResult := fetching.Result{Type: resMetadata.Type, Resource: fetchedResource.GetData()}
 
 	result, err := c.eval.Decision(c.context, fetcherResult)
-
 	if err != nil {
 		c.log.Errorf("Error running the policy: %v", err)
 		return err
 	}
 
+	c.log.Debugf("Eval decision for input: %v -- %v", fetcherResult, result)
```

Comment:
I think this one is going to spam us even on debug. Should we remove it?

Reply:
There is already an equally severe log at debug level from our `libbeat` dependency, which is the full content of each event published to ES. And in the ongoing error scenario, this is exactly the log which helped us to figure out what was going wrong. So I think we can keep it.